runconfig/parse.go

```diff
@@ -513,3 +505,31 @@ func ParseDevice(device string) (DeviceMapping, error) {
 	}
 	return deviceMapping, nil
 }
+
+// ParseVolumesIntoBinds parses any volumes passed by the client and moves any
+// volumes which are in Config.Volumes into HostConfig.Binds.
+func ParseVolumesIntoBinds(c *Config, hc *HostConfig) (*Config, *HostConfig, error) {
```

Comment:
It is left alone - it is exactly that. I honestly don't see your point. 

So again, I ask **what are you asking me to actually do or change**? The only thing (on Windows) that we do with the container path is validate that it is cannot be `c:` or `c:\` as both of those would cause Windows in the container to fail to boot. The same as on Unix, it continues to verify it cannot be `/`. And the other validation is as it previously was to verify that the destination is absolute and `filepath.Clean`ed. I haven't changed any Unix functionality to my knowledge.